pkg/connector/projects.go

```diff
@@ -53,10 +55,11 @@ func (o *projectBuilder) List(ctx context.Context, parentResourceID *v2.Resource
 	var res *gitlabSDK.Response
 	var err error
 
+	groupId := strings.Split(parentResourceID.Resource, "/")[0]
```

Comment:
I think it'd be better to have two helper functions that create/parse group resource IDs instead of splitting/joining strings all over the place. Maybe:

```
func toGroupResourceId(groupId, groupName string) string {...}

// returns group id, group name, or error if it can't parse the string
func fromGroupResourceId(groupResourceId string) (string, string, error) {...}
// or returns group id as an integer if that makes API calls easier?
func fromGroupResourceId(groupResourceId string) (int, string, error) {...}
```